app/containers/scenes/MainMapScene.js

```diff
@@ -88,19 +87,72 @@ export default class MainMapScene extends Component {
 		);
 	}
 
-	_initializeChallengeStartMarkers() {
-		var challenges = mockData.challenges; // TEMP MOCK DATA
-		var headMarkers = this._getHeadMarkers(challenges);
+	_tempMapServerDataToMockData(challenges) {
+
+		challenges.map((challenge, index) => {
+			challenge.id = challenge.id.toString();
+			challenge.locations = challenge.points;
+			challenge.owner = {
+				"id": "0a9ds7a767ac7vr5as4de4465464646",
+				"displayName": "Tina Turner"
+			};
+			challenge.completedBy = [
+				{
+					"id": "1s12fa1243g3kok6800llpmc1q76",
+					"user": "Billy Bob",
+					"message": "Text"
+				}
+			];
+			challenge.created = challenge.dateCreated;
+			return challenge;
+		});
+		return challenges;
+	}
+
+	_loadChallengesForLocation(location, radius) {
+		let longitude = location.longitude;
+		let latitude = location.latitude;
+		var url = "http://enexia.com:10000/geo-challenge/challenge/search?token=geo-ninjas";
```

Comment:
@jwt1143 you're still passing the token as a query param (which I suppose isn't bad because it doesn't change) but I would pass it like the rest of the form data.